project/AppDependencies.scala

```diff
@@ -18,15 +18,17 @@ import sbt._
 
 object AppDependencies {
 
+  val bootstrapPlayVersion = "7.4.0"
+
   val compile = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-backend-play-28"  % "5.24.0"
+    "uk.gov.hmrc"             %% "bootstrap-backend-play-28"  % bootstrapPlayVersion
```

Comment:
Why do you need to create a variable, why can't we just have the version number there?